services/dolly/go-dolly

```diff
@@ -0,0 +1,17 @@
+#!/bin/ash
+
+timestamp=`date +%Y/%m/%d-%H:%M:%S`
+echo "go-dolly: I was called at $timestamp"
+
+config_path="/config/rclone/rclone.conf"
+data_path="/data/test"
```

Comment:
I think this should just be `/data` ?

Maybe `/sync` or something like that would be nicer?

Reply:
Little concerned about doing just `/data` for isoblues that already have 10+gb files... might just have to be vigilante to delete those first

Reply:
Agree with @aaron97neu . Doing `/data/test` just for the demoing purposes. But yes, as @abalmos suggested, `/sync` will be a better idea.

Reply:
`/data` is the path in the container not the host.

Reply:
I think dolly should just sync whatever is at the container's `/data` path. The deploy should put the correct files there. For example, I suspect we may have a compose file like

```yml
services:
   dolly:
      image: isoblue/dolly
      volumes:
         - /path/to/some/data/on/local/fs:/data/mydata
         - can_data:/data/can
         - cell_data:/data/cell
```

Reply:
Got confused with the very similar container and host paths... My bad. I agree with @abalmos this is the way to go. Maybe replacing `data` in the container with `/sync` to ease the confusion